### PR TITLE
Small update on compose files

### PR DESCRIPTION
**Pull request by MichalDulemba**
I added additional info for troubleshooting and compose version for VNC versions of selenium.

### CODE REVIEW

File: basiccompose.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+version: '2'
```
Comment (marconn) [inline basiccompose.txt]: Please change the extension for this file to .yml so it can be used as a base for other users

File: README.md [hunk 2]
```diff
@@ -9,48 +9,29 @@ Docker image to easily run Selenium tests using [webdriverIO](http://webdriver.i
 - [Docker](https://docs.docker.com/engine/installation/)
 
 ## Usage
+Best way to start is clone this repository, go to app/test/specs - you will put your tests there.
+
+Add 
+    before: function() {
```
Comment (marconn) [inline README.md]: Please put this in code context so it can be read easily
```
example
```

File: vnccompose.txt [hunk 3]
```diff
@@ -0,0 +1,38 @@
+version: '2'
```
Comment (marconn) [inline vnccompose.txt]: Please change the extension for this file to .yml so it can be used as a base for other users

**Review by marconn — CHANGES_REQUESTED** [review]
Hi @MichalDulemba, thanks for contributing to these repo
I requested some little changes in order to keep things clean and ready to use
Please check feedback.

File: README.md (hunk 2 above)
Comment (MichalDulemba) [reply README.md]: Moved to another file

**Comment by MichalDulemba** [comment]
Will send another pull request

**Pull request closed without merging**